Tools/Frame_params/RealFlight-quadx.param

```diff
@@ -5,7 +5,7 @@ ARMING_CHECK     0.000000
 ATC_ACCEL_P_MAX  138700
 ATC_ACCEL_R_MAX  168300
 ATC_ACCEL_Y_MAX  11600
-CH7_OPT          4
```

Comment:
I've zapped these real-flight parameter files because they're now held in a separate repo so these realflight related changes can be removed from the PR.

Reply:
OK.